### PR TITLE
Extract ast javascript from javascript package

### LINKED ISSUE #105: AST visitor package
Create an AST visitor and a unique identifier generator for giving unique ids to specific parts of the analysed code.

### PR DESCRIPTION
**Pull request by dstallenberg**
Closes #105

### CODE REVIEW

File: packages/ast-javascript/NOTICE [hunk 1]
```diff
@@ -0,0 +1,5 @@
+SynTest Javascript
```
Comment (mitchellolsthoorn) [inline packages/ast-javascript/NOTICE:1]:
```suggestion
SynTest Framework - SynTest JavaScript
```

File: packages/ast-javascript/package.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@syntest/ast-javascript",
+  "version": "0.0.2-beta.0",
```
Comment (mitchellolsthoorn) [inline packages/ast-javascript/package.json:3]: Did we already release a version of this? I expected the number to be 0.1.0

File: packages/ast-javascript/NOTICE (hunk 1 above)
Comment (dstallenberg) [reply packages/ast-javascript/NOTICE:1]: Fixed.

File: packages/ast-javascript/package.json (hunk 2 above)
Comment (dstallenberg) [reply packages/ast-javascript/package.json]: Fixed.

**Review by mitchellolsthoorn — APPROVED** [review]

**Pull request merged**